trezor-crypto/crypto/secp256k1.table

```diff
@@ -1,1664 +1,1664 @@
 	{
 		/*  1*16^0*G: */
-		{{{0x16f81798, 0x27ca056c, 0x1ce28d95, 0x26ff36cb, 0x070b0702, 0x018a573a, 0x0bbac55a, 0x199fbe77, 0x79be}},
```

Comment:
hmm , why this table changed

Reply:
Interesting, data has changed but results are the same.  Maybe they've optimized the order.